autosklearn/util/backend.py

```diff
@@ -81,9 +82,19 @@ def __init__(self,
             )
         )
         self._output_directory = output_directory
-        self._logger = logging.get_logger(__name__)
+        # We create a dummy logger to start with
+        # Then, when a port is available we can create a
```

Comment:
Could you please rephrase that this is passed from the outside?

Reply:
done!

Reply:
This change is not pushed yet, right?

Reply:
Sorry, I missed adding this filed. It has now been added.